system_modules/napsequenceaudio/src/sequencetracksegmentaudio.h

```diff
@@ -5,7 +5,7 @@
 #pragma once
 
 // NAP Includes
-#include <sequencetracksegment.h>
+#include <sequencetracksegmentduration.h>
```

Comment:
Local include?

Reply:
No because `SequenceTrackSegmentDuration `is part of `napsequence ` and not `napsequenceaudio`, so it's a NAP include